src/test/java/stepDefinition/CategoryPageStepDef.java

```diff
@@ -0,0 +1,17 @@
+package stepDefinition;
+
+import cucumber.api.java.en.When;
+import pages.CategoryPage;
+
+public class CategoryPageStepDef extends AbstractStepDef {
+
+    CategoryPage categoryPage = new CategoryPage(webDriver);
+
+    @When("User clicks on '(.*)' subcategory")
+    public void clickOnSubCategory(String subCategory) {
```

Comment:
could you please add a TODO: to make it generic after implementing **getElementByName()** method, as for you you are not using this parameter

Reply:
Done.
Thank You!